### ISSUE TITLE
CRUD have some issues

### ISSUE THREAD

**Issue by GDemay** [GDemay/aviv-technical-test#10]
We must improve the CRUD system. I'm not 100% that it works

**Comment by GDemay** [comment]
The CRUD is now working #2

**Comment by GDemay** [comment]
A CRUD should perform :

READ, CREATE, UPDATE, DELETE

Not sure that's what I did for Listing

**Comment by GDemay** [comment]
La logique du crud est pas bonne, on fait du create et du update en même temps 

```
    def create(self, listing: Listing):

        if self.get(listing.listing_id) is not None:
            self.update(listing)
            return listing
```

**Comment by GDemay** [comment]
It has been fixed with #26
